src/mscorlib/Resources/Strings.resx

```diff
@@ -2866,9 +2866,6 @@
   <data name="Marshaler_StringTooLong" xml:space="preserve">
     <value>Marshaler restriction: Excessively long string.</value>
   </data>
-  <data name="MemoryDisposed" xml:space="preserve">
```

Comment:
I would keep `MemoryDisposed` and delete `Memory_ThrowIfDisposed` instead. We have `MemoryDisposed` in both CoreRT and CoreFX. `Memory_ThrowIfDisposed` is the CoreCLR-only outlier.